src/templates/admin/core/dashboard.html

```diff
@@ -208,3 +208,9 @@ <h2>Section Editor</h2>
 
     </section>
 {% endblock body %}
+
+{% block js %}
+    {% if published_submissions %}{% include "admin/elements/datatables.html" with target="#published" sort=2 order="desc" %}{% endif %}
+    {% include "admin/elements/datatables.html" with target="#active" sort=2 order="desc" %}
+    {% include "admin/elements/datatables.html" with target="#incomplete" sort=2 order="desc" %}
+{% endblock %}
```

Comment:
There's an `if` statement on one of these but not the others--any reason for that?